Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -479,9 +479,9 @@
 
 workflows[7]  = ['', ['Cosmics','DIGICOS','RECOCOS','ALCACOS','HARVESTCOS']]
 workflows[7.1]= ['', ['CosmicsSPLoose','DIGICOS','RECOCOS','ALCACOS','HARVESTCOS']]
-workflows[7.2] = ['', ['Cosmics_UP17','DIGICOS_UP17','RECOCOS_UP17','ALCACOS_UP17','HARVESTCOS_UP17']]
-workflows[7.3] = ['', ['CosmicsSPLoose_UP17','DIGICOS_UP17','RECOCOS_UP17','ALCACOS_UP17','HARVESTCOS_UP17']]
-workflows[7.4] = ['', ['Cosmics_UP17','DIGICOSPEAK_UP17','RECOCOSPEAK_UP17','ALCACOS_UP17','HARVESTCOS_UP17']]
+workflows[7.2] = ['', ['Cosmics_UP18','DIGICOS_UP18','RECOCOS_UP18','ALCACOS_UP18','HARVESTCOS_UP18']]
+workflows[7.3] = ['', ['CosmicsSPLoose_UP18','DIGICOS_UP18','RECOCOS_UP18','ALCACOS_UP18','HARVESTCOS_UP18']]
+workflows[7.4] = ['', ['Cosmics_UP18','DIGICOSPEAK_UP18','RECOCOSPEAK_UP18','ALCACOS_UP18','HARVESTCOS_UP18']]
```

Comment:
this update disables tests of the 2017 cosmics.
Is it obvious that we will never need to reprocess any 2017 cosmic data?
If not, please re-add the 2017.

Reply:
@slava77 @boudoul what has been the policy so far? Of course we can easily restore the cosmics workflows for 2017, I approved the change as I saw just the last year for these tests. How far in the past would you like to go?

Reply:
Hello @slava77 and @fabiocos , I'm happy to have one 2016 and 2017 , this makes sense - The last suggestion shounds very good to me ('Clone 7.2 for 2017 and 2016 as say 7.12 and 7.22?') I'm happy to make that PR by tomorrow (today is already full ...)

Reply:
I'm not particularly attached to either, but 7.3 (SP is for super-pointing, IIRC) may be a better choice to have something going through the tracker, not just through the muon system

Reply:
@slava77 @boudoul a 2017 workflow can be easily restored, I have a branch with it. But for a 2016 cosmics workflow which conditions should be used? @tocheng @lpernie could you please clarify

Reply:
Thanks @fabiocos , sorry I couldnt make it yesterday, glad that you are looking at it 
- Answering the previous comment from @slava77 : 7.3 is SP , which means that it contains only cosmic tracks going through the pixel volume, while 7.2 is actually also filtering but going through the full tracker volume (strip + pixel) - If we want to have a relevant test of cosmic tracking etc , I would recommend to 'clone' 7.2 for 2017, 2016  . 
I let @tocheng @lpernie  answer you question for 2016 .
Thanks again.